packages/action-merge/src/components/MergeWizard/MergeResponse.js

```diff
@@ -54,6 +52,12 @@ class MergeResponse extends React.Component {
           title={this.props.intl.formatMessage({id: 'client.entityoperation.action.merge.notDeletedEntitiesTitle'})}
           responseEntities={this.props.mergeResponse.notDeletedEntities}
         />
+        {(this.props.mergeResponse.showPermissionMessage)
+          ? <div className="alert alert-info">
+            <FormattedMessage id="client.entityoperation.action.merge.missingReadPermissions"/>
+          </div>
+          : <div/>
```

Comment:
i think the program will behave the same without the empty div in case of `false`?